debezium-operator-commons/src/main/java/io/debezium/operator/commons/OperatorConstants.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.operator.commons;
+
+public final class OperatorConstants {
+
+    public static final String CSV_INTERNAL_BUNDLE_NAME = "${olm.bundle.name}";
```

Comment:
@Naros I suppose you meant this by remark about pulling String into constants? 

The only reason is to have the bundle/grouping (it's a confusing and  overloaded double purpose) name centralised as it is used on 3 different places at the moment.